### DISCUSSION TITLE
My documents not accepted.

### DISCUSSION THREAD

**Discussion by Y-Elghonamy** [community/community#47687] [GitHub Education] [answered]
Hello everyone, 
I have uploaded all papers i have and nothing of it accepted
![image](https://user-images.githubusercontent.com/118995804/219462906-5ce943ab-0465-4e74-98c5-3086cdd8c3d4.png)
Thanks

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @yosefyaser 

There are some answered discussions on this topic, have a look it might help:

[Answered Discussions](https://github.com/community/community/discussions?discussions_q=Your+application+cannot+be+reviewed+until+you+fix+the+following+is%3Aanswered)
